src/task.c

```diff
@@ -670,18 +670,30 @@ JL_DLLEXPORT size_t rec_backtrace_ctx(intptr_t *data, size_t maxsize,
                                       unw_context_t *uc)
 {
 #if !defined(_CPU_ARM_) && !defined(_CPU_PPC64_)
```

Comment:
As mentioned above, my test system triggers abort in llvm similar to #14550 so I can't really check if this is working properly. (The segfault in `rec_backtrace_ctx` is of course "fixed".)